src/applications/vaos/services/mocks/v2/providers.json

```diff
@@ -0,0 +1,38 @@
+{
+  "data": [
+    {
+  "providerIdentifier": 1801312053,
+  "ProviderIdentifierType": "NPI",
+  "name": "ANDERSON, SARA JOE",
+  "ProviderType": "Individual",
+  "Address": "1640 GREGSON ST\r\nDURHAM, NC",
+  "AddressStreet": "1640 GREGSON ST",
+  "AddressCity": "DURHAM",
+  "AddressStateProvince": "NC",
+  "AddressCounty": "DURHAM"
+  
+    },
+    {
+      "providerIdentifier": 1952935777,
+      "ProviderIdentifierType": "NPI",
+      "name": "SOME, PROVIDER",
+      "ProviderType": "Individual",
+      "Address": "1640 GREGSON ST\r\nDURHAM, NC",
+      "AddressStreet": "1640 GREGSON ST",
+      "AddressCity": "DURHAM",
+      "AddressStateProvince": "NC",
+      "AddressCounty": "DURHAM"
+    },
+    {
+      "providerIdentifier": 123,
+      "ProviderIdentifierType": "NPI",
+      "name": "Atlantic Medical Care",
+      "ProviderType": "Individual",
+      "Address": "1640 GREGSON ST\r\nDURHAM, NC",
+      "AddressStreet": "1640 GREGSON ST",
+      "AddressCity": "DURHAM",
+      "AddressStateProvince": "NC",
+      "AddressCounty": "DURHAM"
+    }
+      ]
+}
```

Comment:
Just a suggestion to replace the mock provider api returned test data with names and address that corresponds to the providerNPI in the CC provider selection data (cc_providers.json). That way when we review the confirmation detail page on our local environment, it matches the provider name from the radio selection option.
```suggestion
{
  "data": [
    {
      "providerIdentifier": 1801312053,
      "ProviderIdentifierType": "NPI",
      "name": "AJADI, ADEDIWURA",
      "ProviderType": "Individual",
      "Address": "700 CONSTITUTION AVE NE\r\nDURHAM, NC",
      "AddressStreet": "700 CONSTITUTION AVE NE",
      "AddressCity": "DC",
      "AddressStateProvince": "DC",
      "AddressCounty": "DISTRICT OF COLUMBIA"
    },
    {
      "providerIdentifier": 1952935777,
      "ProviderIdentifierType": "NPI",
      "name": "OH, JANICE",
      "ProviderType": "Individual",
      "Address": "7700 LITTLE RIVER TPKE STE 102\r\nFAIRFAX, VA",
      "AddressStreet": "7700 LITTLE RIVER TPKE STE 102",
      "AddressCity": "ANNANDALE",
      "AddressStateProvince": "VA",
      "AddressCounty": "FAIRFAX"
    },
    {
      "providerIdentifier": 1992228522,
      "ProviderIdentifierType": "NPI",
      "name": "SMAWLEY, DONNA C",
      "ProviderType": "Individual",
      "Address": "5454 WISCONSIN AVE STE 1100\r\nDURHAM, NC",
      "AddressStreet": "5454 WISCONSIN AVE STE 1100",
      "AddressCity": "CHEVY CHASE",
      "AddressStateProvince": "MD",
      "AddressCounty": "CHEVY CHASE"
    }
  ]
}
```